src/pages/[guild]/dashboard/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { Header } from "@/components/Header"
+import {
+  Layout,
+  LayoutBanner,
+  LayoutContainer,
+  LayoutHero,
+  LayoutTitle,
+} from "@/components/Layout"
+import { LayoutHeadline, LayoutMain } from "@/components/Layout"
+import { Center, Heading, Spinner } from "@chakra-ui/react"
+import BackToGuildButton from "components/[guild]/BackToGuildButton"
+import { EditGuildFormComponent } from "components/[guild]/EditGuild/EditGuildFormComponent"
+import DeleteGuildButton from "components/[guild]/EditGuild/components/DeleteGuildButton"
+import { GuildPageBanner } from "components/[guild]/GuildPageBanner"
+import GuildTabs from "components/[guild]/Tabs/GuildTabs"
+import { ThemeProvider } from "components/[guild]/ThemeContext"
+import useGuild from "components/[guild]/hooks/useGuild"
+import useGuildPermission from "components/[guild]/hooks/useGuildPermission"
+import useUser from "components/[guild]/hooks/useUser"
+import Card from "components/common/Card"
+import Section from "components/common/Section"
+import Head from "next/head"
+
+const DashboardPage = () => {
+  const { isDetailed, name } = useGuild()
+  const { isOwner } = useGuildPermission()
+  const { isSuperAdmin } = useUser()
+
+  if (!isDetailed)
```

Comment:
Would be great to handle with NoPermissionToPageFallback if I open the link but don't have proper auth, now we get an infinite loading state in that case